crates/ledger/src/state/volatile_db.rs

```diff
@@ -126,37 +133,25 @@ impl VolatileCache {
 // VolatileState
 // ----------------------------------------------------------------------------
 
-pub struct VolatileState<A> {
-    pub anchor: A,
+#[derive(Default)]
+pub struct VolatileState {
     pub utxo: DiffSet<TransactionInput, TransactionOutput>,
     pub pools: DiffEpochReg<PoolId, PoolParams>,
     pub accounts: DiffBind<StakeCredential, PoolId, Lovelace>,
     pub withdrawals: BTreeSet<StakeCredential>,
     pub fees: Lovelace,
 }
 
-impl Default for VolatileState<()> {
-    fn default() -> Self {
-        Self {
-            anchor: (),
-            utxo: Default::default(),
-            pools: Default::default(),
-            accounts: Default::default(),
-            withdrawals: Default::default(),
-            fees: 0,
-        }
-    }
+pub struct AnchoredVolatileState {
+    pub anchor: (Point, PoolId),
+    pub state: VolatileState,
 }
```

Comment:
(genuine question) Any particular reason to prefer this over a generic (other than aesthetics)?

Reply:
My understanding is that `anchor` will never be something else? Removing the generics clarify that, and simplify a bit the syntax.